src/seedu/addressbook/data/person/Address.java

```diff
@@ -9,10 +9,19 @@
 public class Address {
 
     public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses should be entered in the following format a/BLOCK, STREET, UNIT, POSTAL_CODE";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
+    public static final int INDEX_OF_BLOCK = 0;
```

Comment:
Have you done a check to make sure the user have added the address in the specific format?